### PR TITLE
Fix/fix rest examples

### PR DESCRIPTION
**Pull request by ffiebig**
Fix all rest examples and changed the way docker is built

### CODE REVIEW

File: app/controllers/oneclick_mall_deferred_controller.rb [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+    @amount= @req['amount']
+
+    @resp = Transbank::Webpay::Oneclick::MallDeferredTransaction::refund(
+      buy_order: @buy_order,
+      child_commerce_code: @child_commerce_code,
+      child_buy_order: @child_buy_order,
+      amount: @amount
+    )
+  end
+
```
Comment (afiebig) [inline app/controllers/oneclick_mall_deferred_controller.rb:98]:
```suggestion
```

File: app/views/oneclick_mall_deferred/inscription.html.erb [hunk 2]
```diff
@@ -0,0 +1,17 @@
+<h1>Oneclick Mall Diferido Inscripción</h1>
+
+
+
```
Comment (afiebig) [inline app/views/oneclick_mall_deferred/inscription.html.erb:4]:
```suggestion
```

File: app/views/oneclick_mall_deferred/inscription.html.erb [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+<form method="post" action="/oneclick/mall_deferred/inscription/start" style="display: flex; flex-direction:column; width:50%;font-size: 20px;">
+  <label for="userName">Nombre de usuario</label>
+  <input id="user_name" name="user_name" value="nombre_de_usuario">
+
+  <label for="email">Email</label>
+  <input type="email" id="email" name="email" value="example@example.com">
+
+  <label for="responseUrl">URL de Respuesta</label>
+  <input id="response_url" name="response_url" value="http://localhost:3000/oneclick/mall_deferred/inscription/response_url">
+
```
Comment (afiebig) [inline app/views/oneclick_mall_deferred/inscription.html.erb:14]:
```suggestion
```

File: app/views/oneclick_mall_deferred/inscription_deleted.html.erb [hunk 4]
```diff
@@ -0,0 +1,8 @@
+<h1>Inscripción borrada exitosamente</h1>
+
+<h2>Request</h2>
+<%= @req %>>
+
```
Comment (afiebig) [inline app/views/oneclick_mall_deferred/inscription_deleted.html.erb:5]:
```suggestion
```

File: app/views/oneclick_mall_deferred/inscription_finished.html.erb [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+
+  <label for="parent_buy_order">Orden de compra (comercio padre)</label>
+  <input id="parent_buy_order" name="buy_order" value="<%='12345' + Time.now.to_i.to_s %>"/>
+
+  <label for="details_commerce_code">Detalles de transacción</label>
+  <select id="details_commerce_code" name="detail[details][][commerce_code]" value="<%= @child_commerce_codes.first %>">
+    <option value="<%= @child_commerce_codes.first %>"> Comercio 1 - Codigo <%= @child_commerce_codes.first %></option>
+    <option value="<%= @child_commerce_codes.second %>">Comercio 2 - Codigo <%= @child_commerce_codes.second %></option>
+  </select>
+
```
Comment (afiebig) [inline app/views/oneclick_mall_deferred/inscription_finished.html.erb:26]:
```suggestion
```

File: app/views/oneclick_mall_deferred/inscription_finished.html.erb [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+<form method="post" action="/oneclick/mall_deferred/inscription" style="display: flex; flex-direction:column; width:50%;font-size: 20px;">
+  <input name="_method" type="hidden" value="delete">
+  <label>Nombre de usuario</label>
+  <input name="user_name" value="<%= @user_name %>"/>
+
+  <label>Id de usuario</label>
+  <input name="tbk_user" value="<%= @resp.tbk_user %>"/>
+
+  <button type="submit">Enviar</button>
+
```
Comment (afiebig) [inline app/views/oneclick_mall_deferred/inscription_finished.html.erb:57]:
```suggestion
```

File: app/views/oneclick_mall_deferred/inscription_started.html.erb [hunk 7]
```diff
@@ -0,0 +1,13 @@
+<h1>Oneclick Mall Diferido Inscripción realizada</h1>
+
+
```
Comment (afiebig) [inline app/views/oneclick_mall_deferred/inscription_started.html.erb:3]:
```suggestion
```

File: app/views/oneclick_mall_deferred/transaction_authorized.html.erb [hunk 8]
```diff
@@ -0,0 +1,49 @@
+<h1>Oneclick Mall Diferido transacción autorizada</h1>
+<h1>Request</h1>
+<%= @req %>
+
+
```
Comment (afiebig) [inline app/views/oneclick_mall_deferred/transaction_authorized.html.erb:5]:
```suggestion
```

File: app/views/oneclick_mall_deferred/transaction_authorized.html.erb [hunk 9]
```diff
@@ -0,0 +1,49 @@
...
+<h1>Response</h1>
+<%= @resp.inspect %>
+
+
```
Comment (afiebig) [inline app/views/oneclick_mall_deferred/transaction_authorized.html.erb:9]:
```suggestion
```

File: app/controllers/oneclick_mall_deferred_controller.rb (hunk 1 above)
Comment (ffiebig) [reply app/controllers/oneclick_mall_deferred_controller.rb]: Fixed

**Review by gonzunigad — APPROVED** [review]
LGTM 😄

**Pull request merged**